### DISCUSSION TITLE
Bitwarden/Vaultwarden can’t connect on Windows via Browser-plugin

### DISCUSSION THREAD

**Discussion by buchy01** [dani-garcia/vaultwarden#2417] [General]
Hi,
i need help with my selfhostet Vaultwarden on Synology NAS,I’ll set everything up on the NAS it works on all android devices but not on windows in Browsers such as Chrome, FF, Opera on their plugin Bitwarden.

When i setup Bitwarden connection to my NAS i still get Error mesage: failed to fetch.

any idea why it doesn’t want to work?

Thanks for help.

**Comment by dogeniear** [comment]
If it's self hosted and your on the same ip it won't work, you have to use a proxy or just stay on a vpn
